components/banner/tokenSaleDetails.js

```diff
@@ -0,0 +1,40 @@
+import CountdownHours from '../countdownHours';
+import { Button } from 'antd';
+import Link from 'next/link'
+
+const TokenSaleDetails = ({currentPeriod, secondsUntilNextPeriod, exchangeRate, currentPeriodTotal, active}) => {
+  return (
+    <div className="TokenSaleDetails TokenSaleDetails--is-active">
+      <p className="Banner__title">Period <span className="Banner__title--is-blue">{`#${currentPeriod}`}</span> ends in:
+        <span className="TokenSaleDetails__countdown">
+        {currentPeriod && (
+          <CountdownHours
+            time={secondsUntilNextPeriod}
+            key={currentPeriod}
+          />
+        )}
+        </span>
+      </p>
+      <div className="TokenSaleDetails__group">
+        <div>
+          <p className="TokenSaleDetails__title">Exchange Rate</p>
+          <p className="TokenSaleDetails__value">{`1ETH/${exchangeRate.toLocaleString('en-US', {maximumFractionDigits: 0})} MYB`}</p>
+        </div>
+        <div>
+          <p className="TokenSaleDetails__title">ETH Received</p>
+          <p className="TokenSaleDetails__value">{`${currentPeriodTotal.toLocaleString('en-US')} ETH`}</p>
+        </div>
+      </div>
+      <a href="http://dev-td-mybit.now.sh">
```

Comment:
Please move this to an `ENV` variable. This is going to sneak by our deployment and then we are stuck with people asking on why a `mybit.io` website points to a non-official one. If it weren't our project and I were to see that, I would think the company got hacked or I was about to get pished.

Furthermore, might make sense to add `target="_blank"` to this link, although double check with @cryptnotiq if we want to open a new tab on click or not.

Reply:
Additionally you can make it a constant, but we need to make it obvious where it is so we can change it when it's ready.